### PR TITLE
feat: add description to the application

### PR DESCRIPTION
**Pull request by baptistemesta**
Relates to [RUNTIME-366](https://bonitasoft.atlassian.net/browse/RUNTIME-366)

### CODE REVIEW

**Review by abirembaut — APPROVED** [review]
LGTM

File: community/applications/bonita-super-admin-application.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <applications xmlns="http://documentation.bonitasoft.com/application-xml-schema/1.0">
     <application token="superAdminApp" version="1.0" profile="_BONITA_INTERNAL_PROFILE_SUPER_ADMIN" homePage="user-list" state="ACTIVATED" layout="custompage_BonitaLayoutV8" theme="custompage_themeBonita">
         <displayName>Bonita Super Administrator Application</displayName>
+        <description>Configure and administer the platform (technical user only)</description>
```
Comment (abirembaut) [inline community/applications/bonita-super-admin-application.xml:5]:
```suggestion
        <description>This application allows to configure and administer the platform (technical user only).</description>
```

File: community/bonita-distrib/bonita-super-admin-application.xml [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <applications xmlns="http://documentation.bonitasoft.com/application-xml-schema/1.0">
     <application token="superAdminAppBonita" version="1.0" profile="_BONITA_INTERNAL_PROFILE_SUPER_ADMIN" homePage="user-list" state="ACTIVATED" layout="custompage_layoutBonita" theme="custompage_themeBonita">
         <displayName>Bonita Super Administrator Application</displayName>
+        <description>Configure and administer the platform (technical user only)</description>
```
Comment (abirembaut) [inline community/bonita-distrib/bonita-super-admin-application.xml:5]:
```suggestion
        <description>This application allows to configure and administer the platform (technical user only).</description>
```

File: subscription/applications/bonita-super-admin-application-sp.xml [hunk 3]
```diff
@@ -2,6 +2,7 @@
 <applications xmlns="http://documentation.bonitasoft.com/application-xml-schema/1.0">
     <application token="superAdminApp" version="1.0" profile="_BONITA_INTERNAL_PROFILE_SUPER_ADMIN" homePage="user-list" state="ACTIVATED" layout="custompage_BonitaLayoutV8" theme="custompage_themeBonita">
         <displayName>Bonita Super Administrator Application</displayName>
+        <description>Configure and administer the platform (technical user only)</description>
```
Comment (abirembaut) [inline subscription/applications/bonita-super-admin-application-sp.xml:5]:
```suggestion
        <description>This application allows to configure and administer the platform (technical user only).</description>
```

File: subscription/bonita-distrib/bonita-super-admin-application-sp.xml [hunk 4]
```diff
@@ -2,6 +2,7 @@
 <applications xmlns="http://documentation.bonitasoft.com/application-xml-schema/1.0">
     <application token="superAdminAppBonita" version="1.0" profile="_BONITA_INTERNAL_PROFILE_SUPER_ADMIN" homePage="user-list" state="ACTIVATED" layout="custompage_layoutBonita" theme="custompage_themeBonita">
         <displayName>Bonita Super Administrator Application</displayName>
+        <description>Configure and administer the platform (technical user only)</description>
```
Comment (abirembaut) [inline subscription/bonita-distrib/bonita-super-admin-application-sp.xml:5]:
```suggestion
        <description>This application allows to configure and administer the platform (technical user only).</description>
```

**Review by abirembaut — COMMENTED** [review]
just a suggestion

**Review by pabloalonso — APPROVED** [review]
LGTM

**Pull request merged**